tiling.js

```diff
@@ -2911,16 +2919,19 @@ function insertWindow(metaWindow, {existing}) {
 
     if (!existing) {
         actor.opacity = 0;
+        actor.visible = false;
         clone.x = clone.targetX;
         clone.y = clone.targetY;
-        clone.set_scale(0, 0);
+        clone.set_scale(0, 1);
```

Comment:
I don't particularly care. But do we want to change this? It seems the graphical artifacts are just because of the selection background. Or am I misunderstanding something?

Reply:
With this we stretch the window is just one dimension and I think then I notice the stretching more in the text. But I think it is probably fine since most of the time I am only scrolling left to right this makes it a bit more consistent.